packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -270,6 +272,18 @@ const DatePickerInput = (props) => {
     }
   };
 
+  const handleInvalidInputChange = (val) => {
+    visuallyHiddenComponent.innerText = '';
+    SetInvalidInput(val);
+    if (val) {
+      visuallyHiddenComponent.innerText = intl.formatMessage({ id: 'Terra.datePicker.invalidDate' });
+    }
+  };
+
```

Comment:
```suggestion
```
Using `isInvalid` prop can replace this function and visuallyHiddenText can be updated to 
```js
   <VisuallyHiddenText text={(isInvalid) ? intl.formatMessage({ id: 'Terra.datePicker.invalidDate' }) : '' }/>
```

Reply:
isInvalid prop  is used to highlight the date picker component.
<img width="458" alt="Screenshot 2023-09-08 at 5 03 46 PM" src="https://github.com/cerner/terra-framework/assets/132886427/78482d70-aa15-4487-a603-bfb9686dbd03">